### PR TITLE
Feature/waiter windows

### PR DESCRIPTION
**Pull request by oljamirilovic**

### CODE REVIEW

File: restaurant-front/src/modules/shared/components/view-notifications/view-notifications.component.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ]),
+  ],
+})
+export class ViewNotificationsComponent implements OnInit {
+  @Output() onNotificationsClose = new EventEmitter();
+  @Input() currentUsername: string;
+  @ViewChild('newnotif') tableNew: MatTable<Notification>;
+  @ViewChild('oldnotif') tableOld: MatTable<Notification>;
+  ELEMENT_DATA_NEW: Notification[] = [];
+  ELEMENT_DATA_OLD: Notification[] = [];
```
Comment (RajnovicTeodora) [inline restaurant-front/src/modules/shared/components/view-notifications/view-notifications.component.ts]: I think only constants should be capitalized?

File: restaurant-front/src/modules/waiter/pages/edit-order/edit-order.component.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  @ViewChild(MatSidenav) sidenav!: MatSidenav;
+  @ViewChild(MatTable) table: MatTable<OrderedItem>;
+  totalCost: number;
+  showOrderedItemDetails: boolean;
+  currentOrderedItemDetails: MenuItemWithIngredients;
+  showConfirmAction: boolean;
+  showAdditionalNotes: boolean;
+  currentAdditionalNotes: string;
+  confirmActionTitleDelete: string;
+  ELEMENT_DATA: OrderedItem[] = [];
```
Comment (RajnovicTeodora) [inline restaurant-front/src/modules/waiter/pages/edit-order/edit-order.component.ts]: Should this also be capitalized?

Comment (oljamirilovic) [reply restaurant-front/src/modules/waiter/pages/edit-order/edit-order.component.ts]: Changed to lowerCase

File: restaurant-front/src/modules/shared/components/view-notifications/view-notifications.component.ts (hunk 1 above)
Comment (oljamirilovic) [reply restaurant-front/src/modules/shared/components/view-notifications/view-notifications.component.ts]: Changed to lowerCase

**Pull request merged**